### DISCUSSION TITLE
Welcome to neuron Discussions!

### DISCUSSION THREAD

**Discussion by srid** [srid/neuron#458] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fiatjaf** [comment]
Hello, I don't know how to introduce myself, but neuron is the best note-taking/static site generator I've ever seen in the last 10 years I spent looking for ways to take notes and to make sites -- and awesomely enough it combines both.
